docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -122,3 +122,22 @@ SQL support
 The connector provides :ref:`globally available <sql-globally-available>` and
 :ref:`read operation <sql-read-operations>` statements to access data and
 metadata in Pinot.
+
+.. _pinot-pushdown:
+
+Pushdown
+--------
+
+The connector supports :ref:`aggregate pushdown <aggregation-pushdown>` for the
+following functions:
+
+* :func:`avg`
+* :func:`approx_distinct`
+* :func:`count` and ``count(distinct)``
+* :func:`max`
+* :func:`min`
+* :func:`sum`
+
+Aggregate function pushdown is enabled by default, but can be disabled with the
+catalog property ``pinot.aggregation-pushdown.enabled`` or the catalog session
+property ```aggregation_pushdown_enabled``.
```

Comment:
There is another config to control `count(distinct)` pushdown separately - `pinot.count-distinct-pushdown.enabled` (`count_distinct_pushdown_enabled` is the session property).
Both configs default to true.

And obviously enabling `pinot.count-distinct-pushdown.enabled` without also enabling `pinot.aggregation-pushdown.enabled` doesn't work (there is validation and user gets a helpful error).

This config (to control `count(distinct)`) exists because Distinct count pushdown MAY cause Pinot to do a full scan.

Reply:
Added